content/en/profiler/enabling/java.md

```diff
@@ -204,12 +204,31 @@ For JMC users, the `datadog.MethodSample` event is emitted for wallclock samples
 
 The wallclock engine does not depend on the `/proc/sys/kernel/perf_event_paranoid` setting.
 
-### Datadog profiler allocation engine
+### Profiler allocation engine
 
+{{< tabs >}}
```

Comment:
Suggest switching the order to be consistent with the rest of the tabs on the page. Datadog Profiler tab before JFR tab.

Reply:
The reason I switched the order is that the for allocation profiling we are defaulting to 'JFR' and the user needs to opt in to actually enable the 'Datadog' profiler.
If the order does not make sense, let me know and I will change it - however, it would be nice to make it very explicit that unlike eg. the Datadog CPU profiler, the Datadog allocation profiler is an opt-in.